src/connect.ts

```diff
@@ -29,9 +29,11 @@ export type MutationMapper =
   | string
   | ((this: Vue, commit: Commit, ...args: any[]) => any)
 
+export type StateMapperFunction = (mapper: Function) => (propName: string) => Function;
```

Comment:
```suggestion
export type StateMapperFunction = (mapper: typeof mapState) => Record<string, () => any>
```

You also need to declare a type for mapGetters.